### PR TITLE
BD support fixes

### PR DESCRIPTION
**Pull request by krzyurb**

### CODE REVIEW

File: src/bdSupport/bdSupport.actions.js [hunk 1]
```diff
@@ -112,6 +155,26 @@ class BDSupportActions {
...
+        ts: moment().format('X'),
+      }],
+    });
+
+    const permalinkResponse = await this.slackClient.chat.getPermalink({
+      channel: process.env.BDTECH_ROOM_ID,
+      message_ts: messageResponse.ts,
+    });
+
+    return { message: messageResponse, permalink: permalinkResponse.permalink };
```
Comment (Getriax) [inline src/bdSupport/bdSupport.actions.js]: You are not using the `message` property in the code. Why do you need to return it?

File: src/jira/jira.api.js [hunk 2]
```diff
@@ -37,6 +37,13 @@ class JiraAPI {
     return result;
   }
 
+  async createIssueComment(id, commentBody) {
+    const data = { body: commentBody };
+    const result = await this.jiraRequest(`/issue/${id}/comment`, { data }, 'post');
+
+    return result;
```
Comment (Getriax) [inline src/jira/jira.api.js]: You are not using return value in code, why return it then?

File: templates/bdsupport/description.message [hunk 3]
```diff
@@ -2,11 +2,13 @@
 {{{description}}}
 
 -----------------------
-Category: {{{category}}}
+
 OPP Link: {{{oppLink}}}
```
Comment (Getriax) [inline templates/bdsupport/description.message]: What if `OPP Link` is not specified? Do you want to post it anyway?

File: test/bdSupport/bdSupport.actions.spec.js [hunk 4]
```diff
@@ -147,7 +149,125 @@ describe('bdSupprot actions test suite', () => {
...
+        .post(actionsEndpoint)
+        .send({ payload: JSON.stringify(payload) });
+
+
+      await tick(1);
+
+      expect(response.statusCode).to.equal(200);
+    });
+
+    it('Should respond with status 500 if cannot create dialog', async () => {
```
Comment (Getriax) [inline test/bdSupport/bdSupport.actions.spec.js]: This test case throws an error `ERROR SyntaxError: Unexpected token u in JSON at position 0` this does not look nice, could you fix it? 🙂 If you are expecting this error, could you catch it and check if the proper error was thrown?

File: src/bdSupport/bdSupport.actions.js [hunk 5]
```diff
@@ -21,6 +21,28 @@ class BDSupportActions {
...
   }
 
+  async slashCommandDialog(res, payload) {
+    const user = { id: payload.user_id, name: payload.user_name };
+
+    const userData = await this.slackClient.users.profile.get({ user: user.id });
+    user.email = userData.ok ? userData.profile.email : '';
+
+    const callBackId = 'create-slash-db-support-ticket';
+    const dialog = new DialogBuilder(
```
[the remark this answers is not in the document]
Comment (krzyurb) [reply src/bdSupport/bdSupport.actions.js]: Done

File: templates/bdsupport/description.message (hunk 3 above)
Comment (krzyurb) [reply templates/bdsupport/description.message]: Yes

**Comment by krzyurb** [comment]
@Getriax Fixes done

**Pull request merged**